lib/tests/unit/connector/client_metadata_test.cc

```diff
@@ -34,9 +34,9 @@ static constexpr uint32_t PONG_TIMEOUTS_BEFORE_RETRY { 3 };
 static constexpr uint32_t PONG_TIMEOUT_MS { 10000 };
 
 TEST_CASE("ClientMetadata::ClientMetadata", "[connector]") {
-    SECTION("retrieves correctly the client common name from the certificate (call with proxy constructor)") {
+    SECTION("retrieves correctly the client common name from the certificate (call with proxy, crl constructor)") {
```

Comment:
Do you have a test that shows the connection is rejected because the target is in the CRL?